rtf_vis_tool/yarn.lock

```diff
@@ -0,0 +1,12315 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@alloc/types@^1.2.1":
```

Comment:
this looks auto-generated. do we need to check it in, or is created when the code is run?

Reply:
after reading this [forum](https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files) online, I believe both yarn.lock and package-lock.json files will need to be committed to ensure all users have the same Node dependencies.